src/Resources/npm-package/js/contao-tiny-slider-bundle.js

```diff
@@ -126,6 +137,82 @@ class TinySliderInstance {
 
     }
 
+    modifySliderNav(tinySliderContainer, config) {
+        const navAriaLabel = config.navAriaLabel || false;
+        const dotAriaLabel = config.dotAriaLabel || false;
+        const dotCurrentAriaLabel = config.dotCurrentAriaLabel || false;
+
+        let tnsNav = tinySliderContainer.parentNode.parentNode.parentNode.querySelector('.tns-nav');
+
+        if (tnsNav) {
+
+            this.changeTinySliderNavAriaLabels(tnsNav, navAriaLabel, dotAriaLabel, dotCurrentAriaLabel);
+
+            // Set up an observer
+            const config = {attributes: true, subtree: true};
+
+            const callback = (mutationList, observer) => {
+                for (const mutation of mutationList) {
+                    if (mutation.type === 'attributes' && mutation.attributeName === 'class') {
+                        dotAriaLabel && mutation.target.setAttribute('aria-label', mutation.target.getAttribute('aria-label').replace('Carousel Page', dotAriaLabel));
```

Comment:
Kommst du hier nicht an die Slide-Nummer heran? Dann könnte man das komplette label ersetzen und in der Translation die Position der Nummer individuell bestimmen. Wäre aber nur ein Nice-To-Have